diff --git a/virttest/tests/unattended_install.py b/virttest/tests/unattended_install.py
index 56c5e1f..4d71b8d 100644
--- a/virttest/tests/unattended_install.py
+++ b/virttest/tests/unattended_install.py
@@ -653,6 +653,8 @@ class UnattendedInstallConfig(object):
 
             start_unattended_server_thread(self.unattended_server_port,
                                            self.tmpdir)
+        else:
+            return
 
         # Point installation to this kickstart url
         unattended_file_url = 'http://%s:%s/%s' % (self.url_auto_content_ip,
